src/world/area_kmr/kmr_20/8ED5A0.c

```diff
@@ -8,9 +8,21 @@
 #include "common/foliage.inc.c"
 #define NAMESPACE kmr_20
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8ED5A0", func_80241C9C_8EDABC);
+ApiStatus func_80241C9C_8EDABC(Evt* script, s32 isInitialCall) {
+    func_8011B950(269, -1, 1, 1);
+    set_background_color_blend(0, 0, 0, 255);
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8ED5A0", func_80241CF0_8EDB10);
+    gCameras->bgColor[0] = 0;
```

Comment:
any use of `gCameras->` should actually be `gGameras[CAM_DEFAULT]. applies below as well